src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -34,6 +35,7 @@
     public LogicManager(Model model, Storage storage) {
         this.model = model;
         this.storage = storage;
+        this.undoRedoStack = new UndoRedoStack();
```

Comment:
Probably an unnecessary use of `this`.
```suggestion
        undoRedoStack = new UndoRedoStack();
```